developer_setup.md

```diff
@@ -299,3 +299,32 @@ If this happens, you may be missing developer tools in your OS X. Try to install
 
 If this happens it may be a Bundler issue. Try running `bundle config specific_platform true`
 and re-try.
+
+* Can't install any gems during bundle install
```

Comment:
Is there a more specific error message that presents itself in this scenario you outlined in the PR description? This is a fairly broad statement.